src/ray/object_manager/object_manager.h

```diff
@@ -97,6 +99,72 @@ class ObjectManagerInterface {
   virtual ~ObjectManagerInterface(){};
 };
 
+class PushManager {
+ public:
+  /// Manages rate limiting of outbound object pushes.
+
+  /// Create a push manager.
```

Comment:
Can you add a description of the argument? I.e., is it max chunks per raylet-raylet connection or max chunks across all receivers?